frontend/student/src/routes/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { createFileRoute } from "@tanstack/react-router";
+
+export const Route = createFileRoute("/")({
+  component: RouteComponent,
+});
+
+function RouteComponent() {
+  return <h1>Student app main page</h1>;
+}
```

Comment:
In my opinion, it's better to give this page a significant name, in the long term, it will be multiple index.tsx files and will be difficult to find a specific index.tsx file using the search